keyboards/mlego/m48/rev1/info.json

```
@@ -10,15 +10,111 @@
     "diode_direction": "COL2ROW",
     "encoder": {
         "rotary": [
-            {"pin_a": "A0", "pin_b": "B8"}
+            {
+                "pin_a": "A0",
+                "pin_b": "B8"
+            }
```

```suggestion
            {"pin_a": "A0", "pin_b": "B8"}
```

`qmk format-json` should help here.